src/components/trade/TradesTable/index.tsx

```diff
@@ -0,0 +1,249 @@
+import { OrderSurplusDisplay } from 'components/orders/OrderSurplusDisplay'
+import { constructPrice } from 'utils'
+import { BlockExplorerLink } from 'apps/explorer/components/common/BlockExplorerLink'
+import { COLOURS } from 'styles'
+import { Theme } from 'theme'
+import { variants } from 'styled-theming'
+import TokenImg from 'components/common/TokenImg'
+import { TokenErc20 } from '@gnosis.pm/dex-js'
+
+const { white, blackLight } = COLOURS
+
+const Icon = styled(FontAwesomeIcon)`
+  background: ${({ theme }): string => theme.grey}33; /* 33==20% transparency in hex */
+  border-radius: 1rem;
+  width: 2rem !important; /* FontAwesome sets it to 1em with higher specificity */
+  height: 2rem;
+  padding: 0.4rem;
+  margin-left: 0.5rem;
+  cursor: pointer;
+`
+
+const IconOutline = styled(FontAwesomeIcon)`
+  background: none;
+  border-radius: 1rem;
+  border: 2px solid;
+  width: 2rem !important; /* FontAwesome sets it to 1em with higher specificity */
+  height: 2rem;
+  padding: 0.4rem;
+  margin-left: 0.5rem;
+  cursor: pointer;
+`
+
+export const TableTheme = variants('mode', 'variant', {
+  default: {
+    [Theme.LIGHT]: css`
+      color: ${blackLight} !important;
+    `,
+    [Theme.DARK]: css`
+      color: ${white} !important;
+    `,
+  },
+  get primary() {
+    return this.default
+  },
+})
+
+const TableRow = styled.tr`
+  padding: 6px 2px 6px 2px !important;
+  th {
+    font-weight: 600 !important;
+  }
+`
+const StyledTableRow = styled(TableRow)`
+  /* Fold in theme css above */
+  ${TableTheme}
+`
+
+export const TradesTableHeader = (): JSX.Element => {
+  const tableContext = React.useContext(TradesTableContext)
+
+  return (
+    <StyledTableRow variant={'default'}>
+      <th>
+        Tx Id <IconOutline icon={faQuestion} />
+      </th>
+      <th>Type</th>
+      <th>Surplus</th>
+      <th>Sell Amount</th>
+      <th>Buy Amount</th>
+      <th>
+        Limit price
+        <Icon icon={faExchangeAlt} onClick={tableContext.invertPrice} />
+      </th>
+      <th>
+        Execution Price
+        <Icon icon={faExchangeAlt} onClick={tableContext.invertPrice} />
+      </th>
+      <th>Execution Time</th>
+    </StyledTableRow>
+  )
+}
+
+const TradeTypeWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  span {
+    margin: 5px 0px 5px 0px;
+  }
+`
+
+type TradeTypeProps = {
+  buyToken: TokenErc20
+  sellToken: TokenErc20
+  isLong: boolean
+}
+
+const TradeType = ({ buyToken, sellToken, isLong }: TradeTypeProps): JSX.Element => {
+  const [tokens, setTokens] = React.useState<TokenErc20[]>([])
+  React.useEffect(() => {
+    // isLong - When True, determines if trade type is a BUY else trade type is a SELL
+    setTokens(isLong ? [buyToken, sellToken] : [sellToken, buyToken])
+  }, [isLong, buyToken, sellToken])
+
+  return tokens.length > 0 ? (
+    <TradeTypeWrapper>
+      <span>{isLong ? 'Buy' : 'Sell'}</span>&nbsp;
+      <TokenImg address={tokens[0].address} />
```

Comment:
Token icons should have 16px height and width.

Reply:
The token icons are too small (10px). We need to grow them to 16px.
![image](https://user-images.githubusercontent.com/622217/129786719-b7614afa-2f3a-4b8b-9a5d-9a7f868ae515.png)